example/js/app/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Nit: Whenever there are any files that are generated by `tsc`, you should add the corresponding command line to a build script in package.json. See here for example:

https://docs.microsoft.com/en-us/visualstudio/javascript/compile-typescript-code-npm?view=vs-2022

Reply:
What corresponding command line? Do you mean `"build": "tsc"` under scripts?